tests/actions/PolicyTest.ts

```diff
@@ -84,12 +87,12 @@ describe('actions/Policy', () => {
             });
 
             // Three reports should be created: #announce, #admins and expense report
-            const workspaceReports = _.filter(allReports, (report) => report.policyID === policyID);
-            expect(_.size(workspaceReports)).toBe(3);
-            _.forEach(workspaceReports, (report) => {
-                expect(report.pendingFields.addWorkspaceRoom).toBe(CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD);
-                expect(report.participantAccountIDs).toEqual([ESH_ACCOUNT_ID]);
-                switch (report.chatType) {
+            const workspaceReports = Object.values(allReports ?? {}).filter((report) => report?.policyID === policyID);
```

Comment:
It doesn't look like we need the optional chaining operator. If we do, ignore this comment
```suggestion
            const workspaceReports = Object.values(allReports ?? {}).filter((report) => report.policyID === policyID);
```